google-beta/resource_app_engine_service_split_traffic_test.go

```diff
@@ -0,0 +1,128 @@
+package google
+
+import (
+	"log"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        getTestOrgFromEnv(t),
+		"random_suffix": acctest.RandString(10),
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAppEngineServiceSplitTrafficDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAppEngineServiceSplitTraffic_appEngineServiceSplitTrafficExample(context),
+			},
+			{
+				ResourceName:            "google_app_engine_service_split_traffic.",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"service_id", "split"},
+				ExpectError:             regexp.MustCompile("Resource specified by ResourceName couldn't be found: google_app_engine_service_split_traffic."),
```

Comment:
I don't believe this will get evaluated during an `ImportState` step, only a `Config` one.